sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -252,8 +256,25 @@ public virtual async Task<Response> GetLedgerEntryAsync(string transactionId, st
             scope.Start();
             try
             {
-                using HttpMessage message = CreateGetLedgerEntryRequest(transactionId, collectionId, context);
-                return await _pipeline.ProcessMessageAsync(message, context).ConfigureAwait(false);
+                try
+                {
+                    using HttpMessage primaryMessage = CreateGetLedgerEntryRequest(_ledgerEndpoint, transactionId, collectionId, context);
+                    return await _pipeline.ProcessMessageAsync(primaryMessage, context).ConfigureAwait(false);
```

Comment:
Forgetting a little: does the pipeline account for Not Ready state?  I only remember that the python SDK does not.